draft-ietf-quic-http.md

```diff
@@ -1389,29 +1389,45 @@ for more details.
 #### Initialization
 
 An HTTP implementation MUST NOT send frames or requests which would be invalid
-based on its current understanding of the peer's settings.  All settings begin
-at an initial value, and are updated upon receipt of a SETTINGS frame.  For
-servers, the initial value of each client setting is the default value.
+based on its current understanding of the peer's settings.
+
+All settings begin at an initial value.  Each endpoint MAY use these initial
+values to send messages before the peer's SETTINGS frame has arrived.  When
+the SETTINGS frame arrives, any settings are changed to their new values.
+
+This removes the need to wait for the SETTINGS frame before sending messages.
+Endpoints MUST NOT require any data to be received from the peer prior to
+sending the SETTINGS frame; settings SHOULD be sent as soon as the transport is
+ready to send data.
```

Comment:
I think it's worth adding an Implementation Note along the lines of:
"For clients using a 1-RTT QUIC connection, the 1-RTT keys will always become available prior to SETTINGS arriving, even if the server send SETTINGS immediately.  Clients SHOULD NOT wait indefinitely for SETTINGS to arrive before sending requests, but SHOULD process received datagrams in order to increase the likelihood of processing SETTINGS before sending the first request."